ci-operator/step-registry/cucushift/installer/check/nutanix/customized-resource/cucushift-installer-check-nutanix-customized-resource-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+check_result=0
+
+function check_cpus() {
+   node=$1 cpus=$2
+   vcpuSockets=$(oc get machine -n openshift-machine-api "$node" -o jsonpath='{.spec.providerSpec.value.vcpuSockets}')
```

Comment:
Eventually the setting from install-config.yaml will be passed down to machine itself, machine-api yaml file is just an intermediate step, can we run nutanix client command line to check the setting from the vm instance?

Reply:
fixed, changed to nutanix rest api to check vm instance, pls review again, thanks.